pkg/apis/planetscale/v2/vitesskeyspace_types.go

```diff
@@ -300,12 +300,13 @@ type VitessKeyspaceStatus struct {
 	// This field is only present if the ReshardingActive condition is True. If that condition is Unknown,
 	// it means the operator was unable to query resharding status from Vitess.
 	Resharding *ReshardingStatus `json:"resharding,omitempty"`
-	// FIXME: Convert this to a list for kubectl wait compatibility before merging the feature branch to master.
-	// Conditions is a map of all VitessKeyspace specific conditions we want to set and monitor.
+	// Conditions is a list of all VitessKeyspace specific conditions we want to set and monitor.
 	// It's ok for multiple controllers to add conditions here, and those conditions will be preserved.
-	Conditions map[VitessKeyspaceConditionType]VitessKeyspaceCondition `json:"conditions,omitempty"`
+	Conditions []VitessKeyspaceCondition `json:"conditions,omitempty"`
 }
 
+type VitessKeyspaceConditions = []VitessKeyspaceCondition
```

Comment:
Is this alias used anywhere?

Reply:
Oops, no. I was playing around with trying to see if there was a way to create a custom type with overrides for map style access, which isn't possible in Golang. Missed this artifact - will remove.

Reply:
Removed.